plugins/copy-to-ns.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: copy-to-ns
+spec:
+  homepage: https://github.com/naseemkullah/kubectl-plugins
+  shortDescription: Switch between contexts in your kubeconfig
```

Comment:
> Hi, thanks for submitting this to krew-index. I'm still thinking if `copy-to-ns` is a good name or not. It all depends on the scope you plan for your plugin. Do you intend to add support for other resource types in the future? Otherwise, you can maybe come up with something like `copy-config`. WDYT?

Good point, I have been struggling with the name. Does `copy-config` imply both `secrets` and `configmaps` ?  If so we can keep this plugin for those types and call it a day.

But ultimately yes I do plan to organically (per-demand basis) add more resources if the plugin seems scalable enough (think it is just a matter of just removing the appropriate fields from the in cluster object). I do however see this used mostly for secrets and configmaps, at least for now.

> 
> Also, please support a `--help / -h` parameter in your plugin. This is where people will look for usage first.

Will do.

Reply:
✅